packages/connector-alipay/src/index.ts

```diff
@@ -0,0 +1,227 @@
+/**
+ * The Implementation of OpenID Connect of Alipay Web Open Platform.
+ * https://opendocs.alipay.com/support/01rg6h
+ * https://opendocs.alipay.com/open/263/105808
+ * https://opendocs.alipay.com/open/01emu5
+ */
+import * as crypto from 'crypto';
+import path from 'path';
+
+import {
+  AccessTokenObject,
+  ConnectorError,
+  ConnectorErrorCodes,
+  ConnectorMetadata,
+  GetAccessToken,
+  GetAuthorizationUri,
+  GetUserInfo,
+  ValidateConfig,
+  SocialConnector,
+  GetConnectorConfig,
+  GetTimeout,
+  GetTimestamp,
+} from '@logto/connector-types';
+import { ConnectorType } from '@logto/schemas';
```

Comment:
need to move to `connector-types`, `@logto/schemas` is not expected to be "shared"